js/script.js

```diff
@@ -14,6 +14,8 @@ document.addEventListener("DOMContentLoaded", function(event) {
     xhr.addEventListener("load", function() {
       var response = JSON.parse(xhr.response);
       paragraph.innerHTML = response.value.joke;
+      if (response.value.joke === "" || response.value === undefined)
```

Comment:
your if statement should be before you assign value.joke to paragraph ;) 

you can write it better e.g: `if (!response || !response.value) {...}`